recipe/meta.yaml

```diff
@@ -58,7 +57,7 @@ outputs:
       run_exports:
         # SO name changes at minor rev bumps.
         - {{ pin_subpackage('obake', max_pin='x.x') }}
-        - {{ pin_compatible('boost-cpp', max_pin='x.x') }}
+        - {{ pin_compatible('libboost-devel', max_pin='x.x') }}
```

Comment:
This should not run-export `libboost-devel`, but (if at all) `libboost`. That saves 15k headers for users which won't need it.